api/v1alpha1/condition_consts.go

```diff
@@ -41,6 +41,16 @@ const (
 	VirtualMachineImageNotReadyReason = "VirtualMachineImageNotReady"
 )
 
+const (
+	// PVCHardwareVersionCondition exposes the information of the VirtualMachineImage hardware version
+	// specified in the VirtualMachine.
+	PVCHardwareVersionCondition ConditionType = "PVCHardwareVersion"
```

Comment:
`PVCHardwareVersionSupported` might be a clearer value for this condition (e.g. when you call `conditions.IsTrue(ctx.VM, vmopv1alpha1.PVCHardwareVersionSupportedCondition)`).

Reply:
Refactored the code, the comment doesn't apply to the current version. Resolve conversion.